epochStart/bootstrap/process.go

```diff
@@ -990,7 +981,7 @@ func (e *epochStartBootstrap) updateDataForScheduled(
 		HeadersSyncer:        e.headersSyncer,
 		MiniBlocksSyncer:     e.miniBlocksSyncer,
 		TxSyncer:             e.txSyncerForScheduled,
-		ScheduledEnableEpoch: e.enableEpochs.ScheduledMiniBlocksEnableEpoch,
+		ScheduledEnableEpoch: e.coreComponentsHolder.EnableEpochsHandler().ScheduledMiniBlocksEnableEpoch(),
```

Comment:
why do you still need `ScheduledEnableEpoch` ?

Reply:
the epoch is still used [here](https://github.com/ElrondNetwork/elrond-go/blob/integrate_enable_epochs_handler_p6/epochStart/bootstrap/startInEpochScheduled.go#L223)..decided to only pass the epoch, not the entire component so the `startInEpochWithScheduledDataSyncer` won't be changed